aries-test-harness/features/0025-didcomm-transports.feature

```diff
@@ -0,0 +1,66 @@
+@RFC0025 @AIP20 @UsesCustomParameters @DIDExchangeConnection
+Feature: RFC 0025 DIDComm Transports
+  In order to communicate with other agents,
+  As an Agent
+  I want to various transport protocol.
```

Comment:
@TimoGlastra: something missing in that sentence?

Reply:
Thanks, fixed!